app/views/experiences/card/_show.html.erb

```diff
@@ -10,7 +10,7 @@
     <div class='row'>
       <% if experience.embed.present? %>
         <div class='col-sm-6'>
-          <p><%= experience.description %></p>
+          <p><%= experience.description.html_safe %></p>
```

Comment:
Probably need to do this sort of thing on `app/views/experiences/show.html.erb` as well?